src/test/regress/expected/locally_execute_intermediate_results.out

```diff
@@ -37,11 +37,10 @@ INSERT INTO local_table VALUES                    (3, '3'), (4, '4'), (5, '5'),
 -- prevent PG 11 - PG 12 outputs to diverge
 -- and have a lot more CTEs recursively planned for the
 -- sake of increasing the test coverage
-SET citus.enable_cte_inlining TO false;
```

Comment:
Minor: `prevent PG 11 - PG 12 outputs to diverge` is not relevant anymore

Reply:
Replaced `-- prevent pg 11.*\n` with empty line in the codebase.